### PR TITLE
feat(islamic-website): disclaimer page, member card roles, team assets

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: libs/islamic-website/team-page/src/lib/team-page/team-page.tsx [hunk 1]
```diff
@@ -17,27 +17,96 @@ const Founders = [
...
     image: '/assets/images/founders/hussein-al-meeza.png',
     title: 'Hussein Mohammed Al Meeza',
     description:
       'Hussein Mohammed Al Meeza is an acclaimed award-winning banker with over 45 years of experience spanning the Islamic banking, finance and insurance sectors. A founding member of Emaar properties, Amlak Finance, Emaar Industries & Investments, Emaar Financial services, Dubai Islamic Insurance & Reinsurance Company (AMAN), Nawat, he was named Best Islamic Banking Personality in 2006. His professional career began in 1975 at the Dubai Islamic Bank (DIB), where he spent over 20 years developing the bank’s services, and still serves as Executive Director. As an emerging star of Islamic banking, he is the founder of Al Salam Banks in Sudan, Bahrain and Algeria, where he serves on various positions including Board Member, Vice Chairman and Chairman of the Executive Committee since 2015. He is also the Chairman of Al Salam Bank - Seychelles, Chairman of Top Enterprises L.L.C., Chairman of Lycée Francais Jean Mermoz L.L.C.',
     url: 'http://www.almeeza.com/biography.php',
+    role: 'Co-Founder, Member of the Executive Board',
+  },
+];
+
+const Team = [
```
Comment (olegshilov) [inline libs/islamic-website/team-page/src/lib/team-page/team-page.tsx]:
```suggestion
const team = [
```

File: libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx [hunk 2]
```diff
@@ -15,13 +17,17 @@ export function MemberCard({
...
 }: {
   image: string;
   title: string;
   url?: string;
   onClick?: () => void;
   className?: string;
+  role?: string;
 }) {
+  console.log(title.length);
+
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx]: revert

File: libs/islamic-website/ui-kit/src/lib/text/text.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ export function Text({
     size === 'medium' &&
       'text-[15px] md:text-base leading-[22px] lg:text-[18px] lg:leading-[26px]',
     size === 'large' && 'text-[14px] sm:text-[17px] lg:text-[18px]',
-    isMono ? 'font-mono uppercase' : 'font-serif',
+    isMono ? 'font-vcr uppercase' : 'font-alexandria',
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/text/text.tsx:22]: rtl font

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/islamic-website/team-page/src/lib/team-page/team-page.tsx (hunk 1 above)
Comment (olegshilov) [reply libs/islamic-website/team-page/src/lib/team-page/team-page.tsx]: and `Founders` => `founders`.
use first capital letter only for react compnents and classes

**Review by olegshilov — DISMISSED** [review]

**Review by kioqq — APPROVED** [review]

**Pull request merged**